test/e2e/apps/deployment.go

```diff
@@ -168,6 +173,12 @@ var _ = SIGDescribe("Deployment", func() {
 		e2eskipper.SkipUnlessAtLeast(len(nodes.Items), 3, "load-balancer test requires at least 3 schedulable nodes")
 		testRollingUpdateDeploymentWithLocalTrafficLoadBalancer(f)
 	})
+	ginkgo.It("should not disrupt a load-balancer's connectivity during scaling", func() {
```

Comment:
This test might be a better fit in https://github.com/kubernetes/kubernetes/blob/master/test/e2e/network/loadbalancer.go